### ISSUE TITLE
PeachPie on Android: Could not load assembly 'mvp.xml' during startup registration

### ISSUE THREAD

**Issue by kripper** [peachpiecompiler/peachpie#785]
When running a Peachpie project (latests version) in Xamarin/Android, it crashes on startup with error:
```
Could not load assembly 'mvp.xml' during startup registration
```

Reason:

Peachpie includes `Peachpie.Library.XmlDom -> Mvp.Xml.NetStandard` which throws this error.

Note that Peachpie 0.9.9 works fine on Android because it doesn't include "mvp.xml" assembly.

**Comment by jakubmisek** [comment]
what is the exception message? is it because of missing strong signature, or do you have any other details? stack trace, etc. ?

**Comment by kripper** [comment]
There is no stacktrace. I only found this error in the Android logs: "Could not load assembly 'mvp.xml' during startup registration"
Debugger is not working in my environment (it doesn't attach to the process).
Removing the extension allows the Peachpie app to start normally.

**Comment by jakubmisek** [comment]
We'd need the rest of error messages, usually, there is something like:

```
Could not load assembly 'XXX' during startup registration.
This might be due to an invalid debug instalation.
A common cause is to 'adb install' the app directly instead of doing from the IDE.
```

It might be caused by some 'mvp.xml' dependency.

**Comment by kripper** [comment]
Can you please try to run or debug any Peachpie app on the Android emulator? A simple hello world should fail.

**Comment by jakubmisek** [comment]
if you post a sample app, we can try it

**Comment by kripper** [comment]
Here is a nice test case:
[XamarinAndroidTest.zip](https://github.com/peachpiecompiler/peachpie/files/5147535/XamarinAndroidTest.zip)

**Comment by jakubmisek** [comment]
@kripper I don't have android SDK installed anywhere right now, so maybe someone else might help you.

**Comment by kripper** [comment]
BTW, to easily install the Android SDK from inside MSVS, the recommended servers from Microsoft were not working (tested in two different environments).
The servers from Google worked fine.

**Comment by roberthusak** [comment]
@kripper Hi, I'm unable to reproduce that using the test case. Which version of Android did you use? I tried both Android 9.0 in the emulator (the full name in VS is: "Pixel 2 Pie 9.0 - API 28 (Android 9.0 - API 28)") and a real phone with Android 7.0. Each time I tried both `1.0.0-preview` and `1.0.0-appv4160` (the current latest build by AppVeyor) versions of Peachpie and the app never failed to load. I could even see `mvp.xml` listed in the Modules window in Visual Studio. Updating the Xamarin libraries seems not to cause the problem as well, see the updated solution: [XamarinAndroidTest2.zip](https://github.com/peachpiecompiler/peachpie/files/5317506/XamarinAndroidTest2.zip)
